rules-reviewed/openjdk17/openjdk11/removed-packages.windup.xml

```diff
@@ -27,6 +27,23 @@
                 </hint>
             </perform>
         </rule>
+        <rule id="removed-packages-00010">
+            <when>
+                <or>
+            	    <javaclass references="jdk.nashorn.api.scripting.{*}"></javaclass>
+                    <javaclass references="jdk.nashorn.api.tree.{*}"></javaclass>
```

Comment:
These packages don't match those referenced in the JIRA

Reply:
Which are 
`jdk.scripting.nashorn` and
`jdk.scripting.nashorn.shell`

Reply:
The page linked in the PR mentions
`   jdk.scripting.nashorn -- contains the jdk.nashorn.api.scripting and jdk.nashorn.api.tree packages.
    jdk.scripting.nashorn.shell -- contains the jjs tool.
`

Reply:
The PR covers the first line - what about the jjs tool packages?

Reply:
@m-brophy Thanks for the review. The Jira is referring to modules rather than packages.  The jdk.scripting.nashorn module contains the jdk.nashorn.api.scripting and jdk.nashorn.api.tree packages (the JEP - https://openjdk.org/jeps/372 - provides more information).
Regarding the jjs tool I understand that to a standalone tool rather than anything that would be embedded within an application and hence worth searching for in upgrade rules. WDYT?

Reply:
@m-brophy - I have taken another look. You're right I need to add a rule for jdk.scripting.nashorn.shell module pacakes. I've just found this example  https://code.yawk.at/java/12/jdk.scripting.nashorn.shell/jdk/nashorn/tools/jjs/PackagesHelper.java